serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -1721,6 +1774,16 @@ export class _Exporter {
                                 }
                             }
 
+                            if (babylonNode instanceof Camera) {
+                                // Assign camera index by comparing with node names.
+                                for (let i = 0; i < this._cameras.length; i++) {
+                                    let babylonCamera: Camera = babylonNode;
+                                    if (this._cameras[i].name == babylonCamera.name) {

```

Comment:
I'm not sure this will work all the time. `babylonCamera.name` is not guaranteed to be unique.

Reply:
I use a hashMap to store its index instead. Let me know how it looks to you.